docs/source/utilities.rst

```diff
@@ -0,0 +1,27 @@
+Various utilities
+=================
+
+update_settings
+-----------------
+
+Django provides two utilities for settings overrides: `override_settings` and
+`modify_settings`. The first allows you to simply override a value. The latter
+– to manipulate lists (like `INSTALLED_APPS`). Djet supplements it with
+`update_settings` that allows you to manipulate settings in form of
+dictionaries. The arguments to `update_settings` should be dictionaries with
```

Comment:
They are not only dictionaries but also names of options. IDK how should look precise wording for that in english.